addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -98,16 +98,11 @@ def post(self):
             rec.l10n_latam_document_number = rec.l10n_latam_sequence_id.next_by_id()
         return super().post()
 
-    @api.constrains('name', 'journal_id', 'state')
```

Comment:
@zaoral We should not have removed this line.  (but that does not explain it not working)

Reply:
@zaoral Can you make a PR ASAP?

Reply:
ok

Reply:
the requested PR https://github.com/odoo/odoo/pull/44875